test/controllers/suggested_edit_controller_test.rb

```diff
@@ -56,6 +56,40 @@ class SuggestedEditControllerTest < ActionController::TestCase
     assert_response(400)
   end
 
+  test 'users without the ability to edit posts shouldn\'t be able to approve' do
+    sign_in users(:moderator)
+
+    edit = suggested_edits(:pending_high_trust)
+
+    post :approve, params: { id: edit.id, format: 'json' }
+    assert_response(:bad_request)
```

Comment:
Q: why is this a bad request? Would a comment help or is it just my inexperience?

Reply:
I agree with not doing that here. That's a separate issue and might affect other tests too. Thanks for explaining.

This PR looks good to me by inspection. I'm not currently in a position to test; approving with that caveat and I see Art has also reviewed.